docs/charter.md

```diff
@@ -76,21 +82,16 @@ For details, please consult the [Projects](#projects) list below.
   - The perl interpreter itself
   - Perl Core modules managed by &lt;perl5-porters@perl.org&gt;
 - Supply-chain issues that are found outside of CPAN, for example:
-  - Downstream security issues related to repackaging or patching of CPAN distros by third party packaging system (e.g. Debian's apt, Redhat's rpm, etc).
+  - Downstream security issues related to repackaging or patching of CPAN distros by third party packaging system (e.g. Debian's apt, RedHat's rpm, etc).
 - Third party libraries, files and services linked to or used by CPAN distributions (unless the library is packaged with the affected CPAN distribution);
-- ~~CPAN distributions that function as shims (wrappers) for non-CPAN projects or resources.~~ Examples:
-  - ~~CPAN Distributions that are pure wrappers around compiled libraries (e.g. XS modules wrapping `libxml2` or `sqlite3`)~~
-    - **Rationale**: XS modules are _in scope_ until tooling for warning about upstream vulnerabilities is in place. (sjn)
-  - ~~CPAN Distributions (outside the `Alien::` namespace) that embed or use third-party (non-CPAN) projects or APIs directly.~~
-    - **Rationale**: Shims/wrappers/API modules on CPAN are _in scope_, but we cannot address issues related to the third-party (external) libraries or APIs they link to or use. So for example if `XML::LibXML` contains a vulnerability, it's in scope, but if libxml2 (which it links to) has a vulnerability, we have nothing to do with it, as it is not directly related to code *on* CPAN. (garu)
-  - ~~The `Alien::` distribution namespace~~
-    - **Rationale**: `Alien::` distributions are _in scope_ until tooling for warning about upstream vulnerabilities is in place. (stigo)
-  - ~~Mozilla::CA~~
-    - **Rationale**: `Mozilla::CA` is _in scope_ until the module is either phased out/deprecated or kept automatically in sync with upstream CA. (stigo); Furthermore, Mozilla::CA is apparently about to be auto-updated sometime soon?
 - MetaCPAN
-  - **FIXME**: Offloading management of MetaCPAN security issues depends on getting an agreement with the MetaCPAN folks, which we don't have as of 2023-06-17. Also - is this necessary and desirable? Please share why we should (not) do this.
-- Perl software published outside of CPAN (e.g. Github download links, or with private hosting).
-- …
+   - Handled by the MetaCPAN team.
+- Mozilla::CA
+   - Handled by the libwww-perl team, and semi-automatically updated.
+- Perl software published outside of CPAN (e.g. via Github download links, or with private hosting).
+- Security issues handled by &lt;pause-admin&#64;perl.org&gt;
```

Comment:
I think we can be more generic here and simply say "handled by the PAUSE team"

Reply:
Also, it triggers the same "maybe in conflict with in-scope clause" as MetaCPAN.

Reply:
We did not really talk to the PAUSE team potential for inclusion in CPANSec scope.  As part of the supply chain it is very much in scope though.

Reply:
I still think it's fair to let the PAUSE folks decide if they want us to care, and to what extent. Let's not make a unilateral decision here. They already exist and handle security issues, and if they want to coordinate or need help, I think we'll still be around to have that conversation :smile:

Reply:
My very much in scope is around the supply chain and how we might be able to influence improvements - such as SBOM creation etc.

Reply:
So maybe we should just have _"Facilitating security improvements in the CPAN supply chain"_, drop all mentions to PAUSE & friends and add a specific entry stating something like _"issues related to accounts or services provided by the supply chain (e.g. PAUSE, MetaCPAN, NOC, CPANTesters, etc) are out of scope and handled by their own teams, unless specifically mentioned otherwise"_

Would that make sense?